packages/perseus/src/components/graphie-classes.jsx

```diff
@@ -79,6 +79,7 @@ const createClass = function (spec: $FlowFixMe): $FlowFixMe {
     // React's myDescriptor.constructor.displayName
     GraphieClass.displayName = spec.displayName;
 
+    // $FlowFixMe[invalid-constructor]
```

Comment:
I saw this and was curious why this isn't showing up in webapp.

Reply:
I think I may have copied files over before merging in `master` after Jeff upgraded Flow in webapp.